transformer_engine/common/gemm/cublaslt_gemm.cu

```diff
@@ -109,6 +111,7 @@ void cublas_gemm(const Tensor *inputA,
                                                transb == CUBLAS_OP_N ? k : n,
                                                transb == CUBLAS_OP_N ? n : k,
                                                ldb));
+  NVTE_CHECK_CUBLAS(cublasLtMatrixLayoutCreate(&Cdesc, D_type, m, n, ldd));
```

Comment:
TBH I fail to see why do you have to set C descriptor - it is there for the beta=1 case, right? So it should always be of the same type as D?

Reply:
For FP8 GEMM output, cublas doesn't support FP8 C_type. So we need to set CDesc to bias type for FP8 output and D type for others. Does it make sense?
![image](https://user-images.githubusercontent.com/108759426/211201002-f50ef30c-2ed8-4f2e-9a05-e8bab7b1b5d7.png)